packages/angular_devkit/build_angular/src/utils/process-bundle.ts

```diff
@@ -149,13 +156,21 @@ async function processWorker(options: ProcessBundleOptions): Promise<void> {
       code += `\n//# sourceMappingURL=${path.basename(newFilePath)}.map`;
     }
 
+    if (options.cacheKey && options.cachePath) {
+      const key = options.cacheKey + '-dl-map' + (options.vendorSourceMaps ? '-v' : '');
```

Comment:
We shouldn't be needed to generate different key prefixes for vendor sourcemaps. Check above.

Reply:
Centralized cache key calculation to the parent.